public/locales/en/translation.json

```diff
@@ -31,8 +31,9 @@
     "SCAN QR CODE": "SCAN QR CODE",
     "NAME": "NAME",
     "DATE OF BIRTH": "DATE OF BIRTH",
-    "VACCINATION RECORD": "VACCINATION RECORD"
-  },  
+    "VACCINATION RECORD": "VACCINATION RECORD",
+    "Multiple SMART Health Cards": "Add another SMART\u00AE Health Card for same patient"
```

Comment:
Would stick with the convention of using the string content as the key here, it's easier to read in the code. Also, would change patient to person or individual, since this is a not necessarily being used in a healthcare setting.